packages/shared/src/components/modals/SquadInviteModal.tsx

```diff
@@ -50,15 +50,17 @@ function SquadInviteModal({
           title={`Invite more members to ${squad.name} to stay up to date together and collaborate.`}
         />
       </Modal.Body>
-      <Modal.Footer>
-        <Button
-          icon={<LinkIcon />}
-          className="flex-1 mx-4 btn-primary-cabbage"
-          onClick={() => inviteTextRef.current?.copyLink()}
-        >
-          Copy invitation link
-        </Button>
-      </Modal.Footer>
+      {verifyPermission(squad, SourcePermissions.Invite) && (
```

Comment:
Wait, but we shouldn't even show this modal if you don't have the permission right?
Not sure this step is needed here.

Reply:
@rebelchris It is actually unused so we can remove component altogether?

Reply:
Ah yes, we don't have locked squad anymore, so please do a cleanup yes 🙏